_posts/spells/first-level/2020-10-13-spells-1-arms-of-hadar.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Arms of Hadar
+permalink: "/spells/first-level/arms-of-hadar/"
+layout: spell-post
+categories: [Spells, Cantrips]
```

Comment:
First Level instead of Cantrips for categories

Reply:
Updated all to reflect